cmd/acra-translator/http_api/decryptor_test.go

```diff
@@ -193,11 +242,15 @@ func TestHTTPDecryptionAndResponse(t *testing.T) {
 	if !bytes.Equal(decryptedAcraStruct, []byte("Can't decrypt AcraStruct")) {
 		t.Fatal("Incorrect response body")
 	}
+	if testPoisonCallback.Called {
+		t.Fatal("Callback on poison record wasn't called")
```

Comment:
"Callback on poison record wasn't called" -> "Callback on poison record shouldn't be called"